backend/src/lib/config.ts

```diff
@@ -43,7 +43,7 @@ export async function loadConfigSettings(): Promise<void> {
       }
     }
     for (const key in settings) {
-      if (key.startsWith('LOG_')) {
+      if (key.startsWith('LOG_') || key.startsWith('APP_SEARCH_')) {
```

Comment:
Nit: I guess these are settings that are used in the backend only. A comment here would be good.